edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -53,7 +89,20 @@
 
 <script type="text/javascript">
 
-  var _waiting_for_proctored_interval = null;
+  var accessible_error = function(message, callback) {
+      $("#accessibile-error-modal .cancel-button").click(function(){
+          $("#accessibile-error-modal .close-modal").click();
+      });
+      $("#accessibile-error-modal .ok-button").click(function(){
+          $("#accessibile-error-modal .close-modal").click();
+          callback();
+      });
+
+      accessible_modal("#accessibile-error-modal #confirm_open_button", "#accessibile-error-modal .close-modal", "#accessibile-error-modal", ".content-wrapper");
+      $("#accessibile-error-modal #confirm_open_button").click();
```

Comment:
It is there to simulate a click for the modal code to work. I think it's a weird thing that these modals do specifically.